opm/parser/eclipse/Units/UnitSystem.cpp

```diff
@@ -129,6 +138,9 @@ namespace {
         1, /* gas-oil ratio */
```

Comment:
MSCF/STB is the field unit (metric is sm³/sm³). However, this is the unit transformation **scaling**. At least for metric, since the computation is done in SI (and ratios shouldn't be affected by units), there's no scaling to be done. But since you bring it up, I don't know if we have to scale for field. Lab is scc/scc, and should require no scaling.